api/src/data_migration/copy_oracle_data.py

```diff
@@ -55,19 +55,21 @@ def copy_oracle_data(db_session: db.Session) -> None:
 
     try:
         with db_session.begin():
-            _run_copy_commands(db_session, Schemas.API)
+            _run_copy_commands(db_session, Schemas.API, Schemas.FOREIGN)
```

Comment:
I think we want to make a new script - we don't want to break the v0 opportunity setup.